src/cli/cmds/clone.js

```diff
@@ -0,0 +1,35 @@
+import buildBlueprintCommands from './clone/build-blueprint-commands';
+
+const usage = `Usage:
+  $0 clone <blueprint> <name>
+  $0 help clone <blueprint>`;
+
+module.exports = {
+  command: 'clone <blueprint> <name>',
+  aliases: ['g', 'gen'],
+  describe: 'Clone a blueprint with a different name',
+  builder: yargs => {
```

Comment:
We'll want all commands to follow the same pattern, I'm working to finalize the api for that, but take a look at #132 for the current thinking. @jamesr73 ?